src/pydocstyle/parser.py

```diff
@@ -439,13 +439,20 @@ def parse_definitions(self, class_, all=False):
 
     def parse_all(self):
         """Parse the __all__ definition in a module."""
-        assert self.current.value == '__all__'
+        assert self.current.value == r'__all__'
```

Comment:
Why use raw strings everywhere?

Reply:
Raw strings are processed slightly faster than plain-strings. There are no escapes in my strings, so I can make it a raw string so that the parser does not scan the strings for character escapes.